internal/remediation/suggester/suggest.go

```diff
@@ -0,0 +1,31 @@
+package suggest
+
+import (
+	"context"
+	"fmt"
+
+	"deps.dev/util/resolve"
+)
+
+type SuggestOptions struct {
+	NoMajorUpdates bool
+}
+
+// A VersionSuggester provides an ecosystem-specific method for 'suggesting'
+// the latest version of a dependency based on the given options.
+type VersionSuggester interface {
```

Comment:
Thinking of if/how we can combine the Relaxer and Suggester:
Maybe we can have an `Approach` flag in the options that can be `Relax` or `Latest`, and just decide the behaviour based on that?

On the other hand, I'm not actually sure how much reusable code there is between suggesting latest/relaxing, and if keeping them separate makes it easier to tell when it's not implemented.

Reply:
SGTM!